app/views/insured/family_members/destroyed.js.erb

```diff
@@ -6,6 +6,6 @@ $('#family_error_messages').append("<%= escape_javascript render partial: 'destr
   $('#qle_flow_info').removeClass('hidden');
 <% end %>
 
-<% if ::EnrollRegistry.feature_enabled?(:financial_assistance) &&  (['destroy'].include? controller.action_name)  %>
+<% if ::EnrollRegistry.feature_enabled?(:financial_assistance) &&  (['destroy'].include? controller.action_name) && (@person.active_employee_roles.blank?)  %>
```

Comment:
what if a person has dual role...both employee and consumer.

We should do consumer_role.present? check instead